packages/react/src/Overlay/Overlay.module.css

```diff
@@ -157,12 +157,14 @@
     visibility: hidden;
   }
 
-  &:where([data-variant='fullscreen']) {
-    top: 0;
-    left: 0;
-    width: 100vw;
-    height: 100vh;
-    margin: 0;
-    border-radius: unset;
+  &:where([data-responsive='fullscreen']) {
+    @media screen and (--viewportRange-narrow) {
```

Comment:
Note for reviewer: I changed full screen to only be a responsive thing instead of variant you can apply whenever